Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -1609,12 +1619,15 @@ def revoke_security_group_egress_command(args, aws_client):
     kwargs = {
         'GroupId': args.get('groupId')
     }
+    if args.get('IpPermissionsFull', None):
+        IpPermissions = json.loads(args.get('IpPermissionsFull'))
+        kwargs['IpPermissions'] = IpPermissions
```

Comment:
You can assign the value to the variable inside the if statement. Also when using get() function, if the key does not exist it returns None by default.
Please do it everywhere.
```suggestion
    if IpPermissionsFull := args.get('IpPermissionsFull'):
        IpPermissions = json.loads(IpPermissionsFull)
        kwargs['IpPermissions'] = IpPermissions
```